data-science-onramp/data-ingestion/setup.sh

```diff
@@ -0,0 +1,6 @@
+# Submit a PySpark job via the Cloud Dataproc Jobs API
```

Comment:
I would add a comment at the top with something along the lines of "requires having CLUSTER_NAME and BUCKET_NAME set in your environment"

Reply:
+1